api/v1beta1/types.go

```diff
@@ -84,3 +84,7 @@ type Config struct {
 	// +optional
 	CloudConfig map[string]runtime.RawExtension `json:"cloud-config,omitempty" yaml:"cloud-config,omitempty"`
 }
+
+type ErrorMessage struct {
```

Comment:
🤔 not sure we should putting this here, in the operator controllers api. This is used only in the registration process.
What about moving it in the pkg/register/websocket.go file? After all it is related only to the websocket communication?

Reply:
Yep, that's a better idea! I will change that and also try to add some tests :+1: